src/AppBundle/GitHooks/Action/CheckForBlacklistedWords.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AppBundle\GitHooks\Action;
+
+use CaptainHook\App\Config;
+use CaptainHook\App\Console\IO;
+use SebastianFeldmann\Git\Repository;
+use Symfony\Component\Process\Process;
+
+final class CheckForBlacklistedWords extends Action
+{
+    protected const ERROR_MESSAGE = 'You have left blacklisted keyword in your code!';
+
+    public function execute(Config $config, IO $io, Repository $repository, Config\Action $action): void
+    {
+        if (!$this->isEnabled($action)) {
+            return;
+        }
+
+        $files = $this->getChangedFiles($action, $repository);
+        if (empty($files)) {
+            return;
+        }
+
+        $filesArguments = implode(' ', $files);
+
+        $blacklistedKeywords = $action->getOptions()->get('keyword_blacklist', 'var_dump');
+        $blacklistedArguments = implode('|', $blacklistedKeywords);
+
+        $command = 'grep -iwnHE \''.$blacklistedArguments.'\' '.$filesArguments;
```

Comment:
Using string as the command is deprecated in later Symfony versions. Using array is also valid in Symfony 3.4, so should be used instead of strings.

Reply:
Done in fdd281e478b9f6ccdef8241f15fd8e105bea0524